retriever/engines/postgres.py

```diff
@@ -249,11 +249,16 @@ def get_connection(self):
         import psycopg2 as dbapi
 
         self.get_input()
-        conn = dbapi.connect(host=self.opts["host"],
-                             port=int(self.opts["port"]),
-                             user=self.opts["user"],
-                             password=self.opts["password"],
-                             database=self.opts["database"])
+        try:
+            conn = dbapi.connect(host=self.opts["host"],
+                                 port=int(self.opts["port"]),
+                                 user=self.opts["user"],
+                                 password=self.opts["password"],
+                                 database=self.opts["database"])
+        except dbapi.OperationalError as _:
```

Comment:
Why do we two exception handlers ?

Reply:
I thought if there was another error it would not print the same message

Reply:
I think we can put it on the same line except (dbapi.OperationalError, Exception) as err